scripts/docs_config/introduction.rst

```diff
@@ -0,0 +1,101 @@
+==============
+ Introduction
+==============
+
+Motivation
+==========
+
+The amount of data associated with modern workloads that need to be processed 
+by modern workloads is continuously growing. To address the increasing demand 
+memory subsystem of modern server platforms is becoming heterogeneous. For 
+example, High-Bandwidth Memory (HBM) introduced in Sapphire Rapids addresses 
+throughput needs; the emerging CXL protocol closes the capacity gap and tends 
+to better memory utilization by memory pooling capabilities. Beyond CPU use 
+cases, there are GPU accelerators with their own memory on board.
+The opportunities provided by modern heterogeneous memory platforms come 
+together with additional challenges. This means that additional software 
+changes might be required to fully leverage new HW capabilities. The are two 
+main problems that modern applications need to deal with. The first one is 
+appropriate data placement and data migration between different types of 
+memory. The second one is how SW should deal with different memory topologies.
```

Comment:
`The second one is how SW should leverage different memory topologies.`

Reply:
thanks, done